Mathlib/Topology/Algebra/UniformGroup.lean

```diff
@@ -609,6 +610,18 @@ instance Subgroup.isClosed_of_discrete [T2Space G] {H : Subgroup G} [DiscreteTop
 #align subgroup.is_closed_of_discrete Subgroup.isClosed_of_discrete
 #align add_subgroup.is_closed_of_discrete AddSubgroup.isClosed_of_discrete
 
+@[to_additive]
+lemma Subgroup.tendsto_coe_cofinite_of_discrete [T2Space G] {H : Subgroup G} [DiscreteTopology H] :
+    Tendsto ((↑) : H → G) cofinite (cocompact _) :=
+  IsClosed.tendsto_coe_cofinite_of_discreteTopology inferInstance inferInstance
+
+@[to_additive]
+lemma MonoidHom.tendsto_coe_cofinite_of_discrete [T2Space G] {H : Type _} [Group H] {f : H →* G}
+    (hf : Function.Injective f) (hf' : DiscreteTopology f.range) :
+    Tendsto f cofinite (cocompact _) := by
+  replace hf : Function.Injective f.rangeRestrict := by simpa
+  exact (f.range.tendsto_coe_cofinite_of_discrete).comp hf.tendsto_cofinite
```

Comment:
```suggestion
  exact f.range.tendsto_coe_cofinite_of_discrete.comp hf.tendsto_cofinite
```

Reply:
Somewhat disturbingly, the the brackets are required! I'll raise this on Zulip.

Reply:
Fixed: the issue was that the parameter `H` in `Subgroup.tendsto_coe_cofinite_of_discrete` was incorrectly implicit.